### PR TITLE
feat: add support for unit testing using vitest in editor-ui

### PR DESCRIPTION
**Pull request by alexgrozav**

### CODE REVIEW

File: packages/editor-ui/tsconfig.json [hunk 1]
```diff
@@ -10,11 +10,8 @@
 		"importHelpers": true,
 		"allowSyntheticDefaultImports": true,
 		"baseUrl": ".",
-		"typeRoots": [
```
Comment (netroy) [inline packages/editor-ui/tsconfig.json:13]: are these even needed anymore?
I'm able to remove `typeRoots` and `types`, and still able to build and test everything in this package.

**Review by netroy — COMMENTED** [review]
LGTM. Can you please update the package-lock.json once more? and remove `type` & `typesRoot` from `editor-ui`?

**Comment by alexgrozav** [comment]
@netroy I've updated the `tsconfig.json` file to only include `types`. The `vitest/globals` types are still needed to add automatic global references in unit test files. The `package-lock.json` file has been updated as well.

**Review by netroy — APPROVED** [review]
👍

**Comment by janober** [comment]
Got released with `n8n@0.196.0`

**Pull request merged**